packages/generator-react-server/generators/app/index.js

```diff
@@ -14,6 +14,11 @@ module.exports = yeoman.Base.extend({
 			name: 'name',
 			message: 'What would you like to call your app?',
 			default: this.appname
+		},{
```

Comment:
I ran the branch locally to find #207 and when I verified that it made the existing tests pass, I noticed there was still a linter failure.